modules/cfe_testcase/src/sb_subscription_test.c

```diff
@@ -206,7 +206,7 @@ void TestSBMaxSubscriptions(void)
     while (NumSubs <= CFE_PLATFORM_SB_MAX_MSG_IDS)
     {
         /* fabricate a msgid to subscribe to (this may overlap real msgids) */
-        TestMsgId = CFE_SB_ValueToMsgId(CFE_PLATFORM_CMD_MID_BASE + NumSubs);
+        TestMsgId = CFE_SB_ValueToMsgId(CFE_PLATFORM_CMD_TOPICID_TO_MIDV(NumSubs));
```

Comment:
I believe this is just being used as a sort of "proxy" to establish a valid-ish command msgid.  Basically, it was assuming the base value would be a valid command message ID.

On second look, I don't think this needs to be a command, nor does it even need to be "local" ... It just needs to pass the CFE_SB_IsValidMsgId() test.  So one option might be to just start at one, though this kinda makes other assumptions about the thoroughness of the MsgId checking in this API (or lack thereof).  Thinking that in many cases, "1" won't actually be a usable MsgId value.  But it passes the checks now just because the checks aren't that great.

Reply:
Changed it to simply "1 + NumSubs" ... everything seems to pass.....